platform/pwl.c

```diff
@@ -1960,7 +1986,7 @@ pwl_display_egl_create_buffer_from_image (const PwlDisplay *self,
 
 
 PwlXKBData*
-pwl_display_xkb_get_data (PwlDisplay *self)
+pwl_display_xkb_get_data (PwlSeat *self)
```

Comment:
This operates on a `PwlSeat`, therefore it should be renamed to `pwl_seat_xkb_get_data()`.

Reply:
done.